Description: spelling-error-in-binary usr/lib/libss7.so.1.0 requried required
Bug-Debian: http://lintian.debian.org/tags/spelling-error-in-binary.html

Index: libss7-1.0.2/isup.c
===================================================================
--- libss7-1.0.2.orig/isup.c	2008-08-10 02:54:23.000000000 +1000
+++ libss7-1.0.2/isup.c	2012-06-24 21:33:45.000000000 +1000
@@ -407,7 +407,7 @@
 			hg_str = "ISDN user part not preferred all the way";
 			break;
 		case 2:
-			hg_str = "ISDN user part requried all the way";
+			hg_str = "ISDN user part required all the way";
 			break;
 		case 3:
 			hg_str = "spare";
